cmd/fleetctl/get.go

```diff
@@ -144,6 +144,40 @@ func printHostDetail(c *cli.Context, host *service.HostDetailResponse) error {
 	return printSpec(c, spec)
 }
 
+type EnrichedAppConfigPresenter fleet.EnrichedAppConfig
+
+func (eacp EnrichedAppConfigPresenter) MarshalJSON() ([]byte, error) {
+	type UpdateIntervalConfigPresenter struct {
+		OSQueryDetail string `json:"osquery_detail"`
+		OSQueryPolicy string `json:"osquery_policy"`
+		*fleet.UpdateIntervalConfig
+	}
+
+	type VulnerabilitiesConfigPresenter struct {
+		Periodicity               string `json:"periodicity"`
+		RecentVulnerabilityMaxAge string `json:"recent_vulnerability_max_age"`
+		*fleet.VulnerabilitiesConfig
+	}
+
+	return json.Marshal(&struct {
+		fleet.EnrichedAppConfig
+		UpdateInterval  UpdateIntervalConfigPresenter  `json:"update_interval,omitempty"`
```

Comment:
TIL `json.Marshal` will use this value instead of the `UpdateInterval UpdateIntervalConfig` field within `fleet.EnrichedAppConfig`.